assets/src/edit-story/components/library/panes/shared/libraryMoveable.js

```diff
@@ -202,18 +208,38 @@ function LibraryMoveable({
       } = cloneRef.current.getBoundingClientRect();
       const { x: pageX, y: pageY } = pageContainer.getBoundingClientRect();
 
-      insertElement(type, {
-        ...elementProps,
-        x: editorToDataX(x - pageX, pageSize.width),
-        y: editorToDataY(y - pageY, pageSize.height),
-        width: editorToDataX(w, pageSize.width),
-        height: editorToDataY(h, pageSize.height),
-      });
+      if (type === 'textSet') {
+        insertTextSetByOffset(elements, {
+          offsetX: editorToDataX(x - pageX, pageSize.width),
+          offsetY: editorToDataY(y - pageY, pageSize.height),
+        });
+      } else {
+        insertElement(type, {
+          ...elementProps,
+          x: editorToDataX(x - pageX, pageSize.width),
+          y: editorToDataY(y - pageY, pageSize.height),
+          width: editorToDataX(w, pageSize.width),
+          height: editorToDataY(h, pageSize.height),
+        });
+      }
     }
     resetMoveable();
     return undefined;
   };
 
+  useEffect(() => {
+    if (!editorRef.current) {
```

Comment:
> This looks great, the only thing I see is a scroll bar on the right is added when I hover or drag a text set. Then it goes away on drop.

Looks like this happens due to the Moveable/clone potentially being out of screen while hovering/dragging. Doesn't happen always and in my testing only happens with some screen sizes, that's why I didn't notice it before.

The clone/Moveable are displayed on top of the library/workspace layers since these need to be usable in both library and workspace. This, however, means that for hiding the overflow they cause, it can only be hidden from a very top layer (e.g. the whole editor area). 

Added this `useEffect` which hides the overflow for when clone/Moveable are displayed (hovering/dragging). Tested on different screen sizes (including very small) and this specific addition doesn't seem to break anything.

It should "fix" the issue of scrollbar appearing, however, I'm wondering if we should then completely hide the overflow? Or perhaps there's an alternative instead of using the overflow? @barklund @carlos-kelly LMK if you have any thoughts.

Reply:
Found an alternative meanwhile, reverted this.